solutions/USACO dec15-feast-gold.mdx

```diff
@@ -47,3 +47,60 @@ int main() {
 </CPPSection>
 
 </LanguageSection>
+
+## Solution 2 - Using BFS
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+const int maxn = 5e6 + 5;
+bool pass[maxn];
+
+struct node{
+	int val; bool drunk;
+	node(int _v, bool _d){
+		val = _v;
+		drunk = _d;
+	}

```

Comment:
i think it's not necessary to include these lines

Reply:
It's a little habit of mine, I'll remove it. Thanks for noting!